formbricks/formbricks
# fix: recall info on thank you card

## What does this PR do?

This pr fixes the issue on the Ending card for the Recall Info that was giving undefined when giving placeholder

Fixes # (issue)

[Screencast from 2024-08-16 02-13-59.webm](https://github.com/user-attachments/assets/83ecc48c-241e-497d-8c91-5fac10c181e8)


Fixes [#2992](https://github.com/formbricks/formbricks/issues/2992)

## How should this be tested?

For verifying, you can follow these steps:-

1. Create a survey
2. On Thank you card, without giving attribute to the question page
3. Add recall info for the thank you card
4. It displays the correct fallback value

## Checklist

### Required


### Appreciated
